dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -2887,31 +2881,6 @@ Sedp::received_volatile_message_secure(DCPS::MessageId /* message_id */,
   } else {
     return;
   }
-
-  if (!send_tokens) {
```

Comment:
Can the `send_tokens` variable be removed and functions updated accordingly?

Reply:
Restoring these, since the asymmetric lease stuff didn't work without them.